config/locales/en.yml

```diff
@@ -2357,3 +2357,21 @@ en:
       body_html: Oops! Looks like you're trying to access a page or client you do not have access to. Please contact Partner Support at <a href="mailto:partnersupport@getyourrefund.org">partnersupport@getyourrefund.org</a> if you need to merge a duplicate client you do not have access to.
     home:
       already_started: "Already started? Sign in"
+  confirmation_message:
+    email:
+      subject: Thank you for submitting your tax information
+      body: |
+        Hello %{preferred_name},
+
+        Your tax information has been successfully submitted to your tax preparer at %{org_name}! Your Client ID is %{confirmation_number}.
+
+        Your tax specialist will review your information and will reach out to you in 3-5 business days to review your situation before preparing your taxes.
+
+        You can securely upload any additional tax documents here: %{doc_upload_link}
```

Comment:
I spoke with @yvonnefong, and she was saying we could probably swap this out with the portal login link: "You can check the status of your tax returns and upload additional documents by logging in at https://getyourrefund.org/portal/login". Can you replace the doc upload link with the login link?

Reply:
@bytheway875 definitely! Thanks for the review